app/helpers/events_helper.rb

```diff
@@ -0,0 +1,2 @@
+module EventsHelper
```

Comment:
Blank `EventsHelper` can be safely removed. Consider adding a `g.helpers = false` option to the generators section in `config/application.rb` (similar to https://github.com/xudongcamsys/homefoodie/pull/8/files#r33441767 ) to stop helpers being generated.